products/vmware_cb_enterprise_edr.py

```diff
@@ -96,7 +105,8 @@ def divide_chunks(self, l: list, n: int) -> Generator:
             yield l[i:i + n]
 
     def perform_query(self, tag: Tag, base_query: dict, query: str) -> set[Result]:
-        results = set()
+ 
+        results = list()
```

Comment:
If I recall correctly, this product can return duplicate process information which is why `set()` was used. Were there errors that required it to be replaced with `list()`?

Reply:
Yeah it was to account for the raw_results functionality which is dormant at the moment. There were some errors occuring. Going to reset to `set` and test raw further for the next update.

Reply:
```
raw_results = list()
if self._raw:
    return raw_results  
else:  
    return results
```
Went with this instead to mirror CbR, and return everything directly from the CBC API (No post-processing).

Reply:
Disregard maintained set for both.

Reply:
Disregard again. I need that to be it's own list because deets is a dict and I can't add a dict to a set.